src/test/kotlin/io/kotest/plugin/intellij/psi/SpecTests.kt

```diff
@@ -21,51 +21,6 @@ class SpecTests : LightJavaCodeInsightFixtureTestCase() {
       return false
    }
 
-   fun testIsContainedInSpecFunSpec() {
```

Comment:
Instead of merging `master´, rebase to keep the PR's history clean of merge commits for more convenient review.